src/components/ListingCard/styles.tsx

```diff
@@ -1,32 +1,44 @@
 import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { Open_Sans as OpenSans } from 'next/font/google';
+import { H4 } from '@/styles/text';
+
+const openSans = OpenSans({ subsets: ['latin'] });
 
 // the card itself
 export const CardBody = styled.div<{ $selected?: boolean }>`
```

Comment:
question: do we want the listing card to take on different heights depending on how big the tags are? seems like that's what we're doing rn, but it potentially feels cleaner to always have them be a certain height (cc @davidqing6432)

Reply:
Solved for now? Talked to @davidqing6432 during worksesh today, and decided on increasing the width of the listing cards to make the size difference less obvious. Let me know if we wanna go for a different approach though!!